src/EditorFeatures/Core/Implementation/IntelliSense/AsyncCompletion/ItemManager.cs

```diff
@@ -239,6 +248,43 @@ internal ItemManager(RecentItemsManager recentItemsManager)
                 highlightedList,
                 completionHelper,
                 hasSuggestedItemOptions);
+
+
+            static bool ShouldBeFilteredOutOfCompletionList(VSCompletionItem item, ImmutableArray<CompletionFilter> activeNonExpanderFilters)
```

Comment:
>static [](start = 12, length = 6)

please specify that it is private or internal

Reply:
These are local functions. I guess using camelCase would help in this case :) (#35944)

Reply:
Makes sense. nit: Do we need static for local functions?

---
In reply to: [296038163](https://github.com/dotnet/roslyn/pull/36146#discussion_r296038163) [](ancestors = 296038163)